nvflare/fuel/hci/zip_utils.py

```diff
@@ -39,12 +40,29 @@ def _path_join(base: str, *parts: str) -> str:
     return path.replace("\\", "/")
 
 
-def remove_leading_dotdot(path):
+def remove_leading_dotdot(path: str) -> str:
     while path.startswith("../"):
         path = path[3:]
     return path
 
 
+def split_path(path: str) -> (str, str):
```

Comment:
Do we need to consider if the path is only "/"?

Reply:
it would be better just write a simply unit tests